File: not-very-secure/not-very-secure.js
function alphanumeric(string) {
    if(string === "") return false;
    const checkSpaces = /[\s]/g.test(string);
    const checkSpecialChars = /[-_!{}()"£$%^&*¬`,.<>/@~#=+]/g.test(string);
    if(checkSpaces || checkSpecialChars) return false;
    else return true;
};

module.exports = { alphanumeric };

//https://www.codewars.com/kata/526dbd6c8c0eb53254000110/train/javascript

File: python-venv/not_very_secure/kata_function/not_very_secure.py
import re

def alphanumeric(string):
    if string == "":
        return False
    elif len(re.findall("\s", string)) > 0:
        return False
    elif len(re.findall("[-_!{}()\"£?$%^&*¬'``,.<>/@~\[\]#=+:\;\|\\\\]", string)) > 0:
        return False

    return True

#https://www.codewars.com/kata/526dbd6c8c0eb53254000110/train/python